composer.json

```diff
@@ -36,7 +36,6 @@
 		"codeception/module-filesystem": "^1.0",
 		"codeception/module-cli": "^1.0",
 		"codeception/util-universalframework": "^1.0",
-		"automattic/vipwpcs": "^2.2",
```

Comment:
We are not currently checking the VIP Go ruleset. So I'm removing this unused package for now.